python/runtime-protect/custom_llm_metric_protect_test/utils/list_metrics.py

```diff
@@ -0,0 +1,91 @@
+"""
+Verify that a custom metric was created successfully
+
+This helper probes the installed `galileo` package for scorer/metric listing
+APIs and prints manual verification steps if programmatic listing isn't available.
+"""
+
+import os
+from dotenv import load_dotenv
+
+# Load environment variables
+load_dotenv()
+GALILEO_API_KEY = os.getenv("GALILEO_API_KEY")
+
+if not GALILEO_API_KEY:
+    print("❌ GALILEO_API_KEY not set. Please add it to .env")
+    exit(1)
+
+try:
+    import galileo
+except Exception as e:
```

Comment:
Do we need this? I know it can be good practice to verify package installations, but my preference is simple clean code that assumes the user has installed the right things. Error handling for examples can be noise.